administration/src/bp-modules/applications/VerificationsView.tsx

```diff
@@ -68,6 +73,24 @@ export const getStatus = (verification: Application['verifications'][number]) =>
   }
 }
 
+export const JulicaVerificationQuickIndicator = memo(() => {
+  return (
+    <Tooltip
+      content={
+        <div>
+          <b>Bestätigung(en) durch Organisationen:</b>
+          <br />
+          Bestätigung ist nicht erforderlich
+        </div>
+      }>
+      <UnFocusedDiv>
+        <Indicator status={VerificationStatus.Verified} />
+        <img src={JuleicaLogo} alt='juleica' height='100%' />
+      </UnFocusedDiv>
+    </Tooltip>
+  )
+})
+
 export const VerificationsQuickIndicator = memo(
```

Comment:
this can also be exported to a separate component. Sorry for all the refactoring issue!